hooks/nvidia-bootstrap/image/Dockerfile

```diff
@@ -12,9 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM alpine:3.6
+FROM debian:jessie
```

Comment:
Why the switch?  We should probably use the base k8s container ... see protokube

Reply:
nevermind I see why you switched.  We should probably use the same container as protokube.

Reply:
Must have or should have?  The difference is I've already got a lot of mileage on the existing base image.

Reply:
We're switching away from alpine generally, AIUI, so +1 to debian.